coss/base/templates/base.html

```diff
@@ -7,6 +7,11 @@
     <meta http-equiv="Content-type" content="text/html; charset=utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
 
+    {% compress js %}
+      <script type="text/javascript" src="{% static 'lib/dnt-helper/js/dnt-helper.js' %}"></script>
+      <script type="text/javascript" src="{% static 'js/google-analytics.js' %}"></script>
```

Comment:
Is this still used? AFAIK the gtm.js is the only dependency needed, or am I wrong?

Reply:
This is our own js [source code](https://github.com/mozilla/coss/blob/master/coss/base/static/js/google-analytics.js) for analytics and gtm. Or did I misread your question?

Reply:
Ehm, yes, of course.. Sorry